### PR TITLE
docs: capitalize webpack with a lowercase W

### PR DESCRIPTION
**Pull request by ugultopu**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: CHANGELOG_ARCHIVE.md
Comment (bencodezen) [inline CHANGELOG_ARCHIVE.md:1]: This file should not be part of the change.

File: .yarn/releases/yarn-1.22.19.cjs
Comment (bencodezen) [inline .yarn/releases/yarn-1.22.19.cjs:1]: This file should not be part of the change.

**Review by bencodezen — CHANGES_REQUESTED** [review]
Thanks for the PR @ugultopu!

I'm totally on board with having the webpack brand be capitalized correctly, but we have a couple of files that are unnecessary for the change.

Once you've updated the PR and squashed the commits, happy to approve so this can get merged. Thank you!

**Comment by ugultopu** [comment]
Thank you Ben, updated!

**Comment by JeanMeche** [comment]
Can you also take care of reverting the 2 files that should not be altered, thank you.

**Comment by ugultopu** [comment]
> Can you also take care of reverting the 2 files that should not be altered, thank you.

Currently they just have whitespace changes, which is the result of my text editor removing trailing whitespaces. I think those changes are good since some lines had trailing whitespace but others didn't have, leading to an inconsistency.

I can still revert though.

**Comment by JeanMeche** [comment]
The changelog archive is not meant to be ammended and `.yarn/releases/yarn-1.22.19.cjs` is akin to a binary and shouldn't be changed manually. 
We'd prefer to have the changes removed.

**Comment by ugultopu** [comment]
Updated 👍

**Review by JeanMeche — APPROVED** [review]
Thank you for the changes

**Review by bencodezen — APPROVED** [review]
Thanks for making the changes and the PR @ugultopu!

**Review by josephperrott — APPROVED** [review]
LGTM

**Review by alxhub — APPROVED** [review]
Reviewed-for: global-approvers

**Comment by AndrewKushnir** [comment]
@ugultopu it looks like the change in this PR actually lowercases "W" in all places where Webpack is mentioned. Could you please confirm is this is the expected change and if yes, add more info into a commit message (to explain why we did that)? Thank you.

**Comment by JeanMeche** [comment]
We discussed that during triage wednesday, this is part of webpack's brand guidelines https://webpack.js.org/branding/

**Comment by AndrewKushnir** [comment]
@JeanMeche thanks for additional info 👍 It'd be great to add this info into the commit message for future context.

**Comment by thePunderWoman** [comment]
@ugultopu Looks like this has conflicts. Can you resolve those and we'll get it landed?

**Comment by thePunderWoman** [comment]
Caretaker Note: this is a docs only change and is save to merge without a presubmit.

**Comment by thePunderWoman** [comment]
This PR was merged into the repository by commit 0f1c71869e522f5a49edba80ba660c231f62a3e6.

The changes were merged into the following branches: main, 19.0.x
